pipeline/dbt/models/staging/sources/mission_locale/stg_mission_locale__structures.sql

```diff
@@ -0,0 +1,27 @@
+WITH source AS (
+    {{ stg_source_header('mission_locale', 'structures') }}
+),
+
+final AS (
+    SELECT
+        _di_source_id                                                             AS "_di_source_id",
+        NULLIF(TRIM(data -> 'structures' ->> 'id_structure'), '')                 AS "id_structure",
+        NULLIF(TRIM(data -> 'structures' ->> 'email'), '')                        AS "email",
+        NULLIF(TRIM(data -> 'structures' ->> 'commune'), '')                      AS "commune",
+        NULLIF(TRIM(data -> 'structures' ->> 'siret'), '')                        AS "siret",
+        NULLIF(TRIM(data -> 'structures' ->> 'horaires'), '')                     AS "horaires",
+        NULLIF(TRIM(data -> 'structures' ->> 'site_web'), '')                     AS "site_web",
```

Comment:
'https://' || regexp_replace(site_web, 'https?://', '')
```suggestion
        'https://' || REGEXP_REPLACE(NULLIF(TRIM(data -> 'structures' ->> 'site_web'), ''), 'https?://', '')                     AS "site_web",
```